nebu/_version.py

```diff
@@ -505,6 +529,10 @@ def get_versions():
 
     try:
         pieces = git_pieces_from_vcs(cfg.tag_prefix, root, verbose)
+        latest_version = get_latest_released_version()
+        if pieces["closest-tag"] and pieces["closest-tag"] < latest_version:
```

Comment:
Passing the response value structure from pypi.org through a series of functions seems like a way to get lost quickly if for some reason they decide to change their api schema, which is likely since this newer pypi.org service is still in beta.

What if we pull out the pieces we need just after the request? This would likely enable us to better infer that a problem is with the request data not the results of our functions.